src/common/k8s-api/endpoints/ingress.api.ts

```diff
@@ -196,6 +198,24 @@ export class Ingress extends KubeObject {
   }
 }
 
+export function computeRuleDeclarations(ingress: Ingress, rule: IngressRule): ComputedIngressRoute[] {
+  const { host = "*", http: { paths } = { paths: [] }} = rule;
+  const protocol = (ingress.spec.tls?.length ?? 0) > 0
```

Comment:
I'm seeing every link getting https, something fishy here?

Reply:
Thanks for catching this.